plugins/source/aws/docs/tables/README.md

```diff
@@ -568,7 +568,11 @@
 - [aws_ssm_sessions](../../../../../website/tables/aws/aws_ssm_sessions.md)
 - [aws_ssoadmin_instances](../../../../../website/tables/aws/aws_ssoadmin_instances.md)
   - [aws_ssoadmin_permission_sets](../../../../../website/tables/aws/aws_ssoadmin_permission_sets.md)
+    - [aws_ssoadmin__customer_managed_policies](../../../../../website/tables/aws/aws_ssoadmin__customer_managed_policies.md)
     - [aws_ssoadmin_account_assignments](../../../../../website/tables/aws/aws_ssoadmin_account_assignments.md)
+    - [aws_ssoadmin_inline_policies](../../../../../website/tables/aws/aws_ssoadmin_inline_policies.md)
```

Comment:
Thoughts on naming these aws_ssoadmin_permission_set_inline_policies since the policies are part of the permission sets?

Also applies for the other tables in this PR

Reply:
I see where you are coming from, but should we also include the `_instance_` in the name, because it is all part of an sso instance?

Reply:
I see your point, but ultimately don't think instance belongs here.  

* Inline Policies could be aws_ssoadmin_permission_set_inline_policies.
* Permission Boundaries could be aws_ssoadmin_permission_set_permissions_boundaries

A couple options: 

1.  For the other policies, we could either follow the same convention as inline policies.

2.  We could go by attachment such as:
* aws_ssoadmin_permission_set_managed_policy_attachments
* aws_ssoadmin_permission_set_customer_managed_policy_attachments.

3.  Another option is to mirror what we have for IAM resources such as aws_iam_role_attached_policies and do something like aws_ssoadmin_permission_set_attached_managed_policies

This gets tricky depending on how close we want to follow the AWS Identity Center Model.  https://docs.aws.amazon.com/singlesignon/latest/APIReference/API_Operations.html

As of right now, I don't have a clear favorite option.

Reply:
I am fine with 
- `aws_ssoadmin_permission_set_inline_policies`
- `aws_ssoadmin_permission_set_customer_managed_policies`
- `aws_ssoadmin_permission_set_managed_policies`